nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -1208,8 +1209,12 @@ void CALLBACK GeckoVBufBackend_t::renderThread_winEventProcHook(HWINEVENTHOOK ho
 			continue;
 		LOG_DEBUG(L"found active backend for this window at "<<backend);
 
-		//For focus and alert events, force any invalid nodes to be updated right now
-		if(eventID==EVENT_OBJECT_FOCUS||eventID==EVENT_SYSTEM_ALERT) {
+		//For focus, documentLoadComplete and alert events, force any nodes already marked as invalid  to be updated right now,
+		if(
+			eventID == EVENT_OBJECT_FOCUS
+			|| eventID == IA2_EVENT_DOCUMENT_LOAD_COMPLETE
+			||eventID==EVENT_SYSTEM_ALERT

```

Comment:
nit: space after boolean OR 
```suggestion
			|| eventID==EVENT_SYSTEM_ALERT

```